bigbluebutton-tests/playwright/core/page.js

```diff
@@ -154,11 +154,21 @@ class Page {
     await expect(locator).toBeHidden({ timeout });
   }
 
+  async wasNthElementRemoved(selector, count, timeout = ELEMENT_WAIT_TIME) {
+    const locator = this.getLocator(':nth-match(' + selector + ',' + count + ')');
+    await expect(locator).toBeHidden({ timeout });
+  }
+
   async hasElement(selector, timeout = ELEMENT_WAIT_TIME) {
     const locator = this.getLocator(selector);
     await expect(locator).toBeVisible({ timeout });
   }
 
+  async hasNElements(selector, count, timeout = ELEMENT_WAIT_TIME) {
```

Comment:
Could we use `locator.count()` for this? (https://playwright.dev/docs/api/class-locator#locator-count)